src/Pdoxcl2Sharp/Globals.cs

```diff
@@ -8,7 +8,7 @@ namespace Pdoxcl2Sharp
 {
     public static class Globals
     {
-        public static readonly Encoding ParadoxEncoding = CodePagesEncodingProvider.Instance.GetEncoding(1252);
+        public static readonly Encoding ParadoxEncoding = Encoding.UTF8;
```

Comment:
- afaik, the bom isn't present, so we should keep that same behavior when saving.
- And can you preserve the default windows-1252. So we aren't adding a feature and breaking change in the same release